README.md

```diff
@@ -453,6 +453,20 @@ https://developers.google.com/analytics/devguides/collection/protocol/v1/paramet
 
 Staccato provides a variety of adapters for sending or debugging requests being made. To use them, first require the adapter by name: `require 'staccato/adapter/#{chosen-adapter-name}'`
 
+Multiple adapters can be used by calling `add_adapter`:
+
+```ruby
+require 'staccato/adapter/validate'
+
+tracker = Staccato.tracker('UA-XXXX-Y') do |c|
+  c.add_adapter Staccato::Adapter::Validate.new
+  c.add_adapter Staccato::Adapter::Logger.new(Staccato.ga_collection_uri)
+  c.add_adapter Staccato::Adapter::Faraday.new(Staccato.ga_collection_uri)
+end
+```
+
+**Results returned will be in an array, where each response is a map to each adapter**
```

Comment:
This is a bit unclear. From the source, it like like the results will simply be an array, and your use of "map" here makes me think of a hash. Where really, it will be an array of responses in the same order as the adapters were added.

What about:
"Results returned will be in an array, as returned by each adapter in the order the adapters were added".

Another option would be to actually return a hash, the keys being adapters, and the values the results. But that might be getting a bit too complicated.